libmscore/score.h

```diff
@@ -1086,6 +1108,7 @@ class Score : public QObject, public ScoreElement {
       virtual QVariant propertyDefault(P_ID) const override;
 
       virtual inline QQueue<MidiInputEvent>* midiInputQueue();
+      virtual inline QLinkedList<MidiInputEvent>* activeMidiPitches();
```

Comment:
I thought a linked-list would give better performance in this situation because pitches will be added and removed all the time so the list needs to change in size. Also, although I always add pitches to the end of the list, I will be removing them from anywhere in the list and this is faster with a linked-list. Finally, I only need to access the list in-order, I never need immediate access to a random pitch in the middle, so the usual performance penalties for linked-lists don't apply.